libusb/os/linux_usbfs.c

```diff
@@ -95,6 +95,9 @@ static int sysfs_available = -1;
 /* how many times have we initted (and not exited) ? */
 static int init_count = 0;
 
+/* is no authority to operate usb device directly */
+static int weak_authority = 0;
+
```

Comment:
Maybe the code should use `#ifdef __ANDROID__` around the declaration since `weak_authority` is only available for Android in the `op_set_option()` function?

This is to avoid dead code in non-Android version of the lib.

Reply:
Agreed